invitaions.go

```diff
@@ -0,0 +1,37 @@
+package mackerel
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Invitation information
+type Invitation struct {
+	Email     string `json:"email,omitempty"`
+	Authority string `json:"authority,omitempty"`
+	ExpiresAt int64  `json:"expiresAt,omitempty"`
+}
+
+// FindInvitations find invitations.
+func (c *Client) FindInvitations() ([]*Invitation, error) {
+	req, err := http.NewRequest("GET", c.urlFor(fmt.Sprintf("/api/v0/invitations")).String(), nil)
```

Comment:
Why this uses `fmt.Sprintf`?
I guess because other functions such as *FindUsers* is used `fmt.Sprintf`, but I think it's not needed.

```go
c.urlFor("/api/v0/invitations").String()
```

Reply:
I mistook this and modified.